source/extensions/priv/server/ntds_decrypt.c

```diff
@@ -0,0 +1,129 @@
+#include "precomp.h"
+
+void bytes_to_string(LPBYTE data, int length, LPSTR output){
+	for (int i = 0; i < length; i++){
+		sprintf(output + (i << 1), "%02X", ((LPBYTE)data)[i]);
+	}
+}
+
+BOOL decrypt_hash(encryptedHash *encryptedNTLM, decryptedPEK *pekDecrypted, char *hashString, DWORD rid){
+	BOOL cryptOK = FALSE;
+	BYTE encHashData[17] = { 0 };
+	BYTE decHash[17] = { 0 };

```

Comment:
Would it be possible to get the `16` and `17` values in this function replaced with some constants please?